package.json

```diff
@@ -73,6 +74,7 @@
     "yargs-parser": "^21.1.1"
   },
   "devDependencies": {
+    "@transcend-io/cli": "^4.72.0",
```

Comment:
```suggestion
```

added by accident? when running locally, you can run `yarn build` to make the cli command accessible